src/main/java/com/hackoeur/jglm/Mat4.java

```diff
@@ -257,23 +257,23 @@ public int getNumRows() {
 	public int getNumColumns() {
 		return 4;
 	}
-	
+
 	@SuppressWarnings("unchecked")
 	@Override
 	public <T extends Vec> T getColumn(final int columnIndex) {
 		assert columnIndex < 4 : "Invalid column index = " + columnIndex;
-		
+
 		switch (columnIndex) {
-		case 0:
-			return (T) new Vec4(m00, m01, m02, m03);
-		case 1:
-			return (T) new Vec4(m10, m11, m12, m13);
-		case 2:
-			return (T) new Vec4(m20, m21, m22, m23);
-		case 3:
-			return (T) new Vec4(m30, m31, m32, m33);
-		default:
-			throw new IllegalArgumentException("Invalid column index = " + columnIndex);
+			case 0:
```

Comment:
Well, in this particular instance, just as it was before this change.  In general, indent size = 4 using tabs.  If you are using Eclipse then I think the default formatter config should be fine.  As for the other parts of the diff involving whitespace, I think just leaving things formatted as they were before would be fine.